components/Navigation.js

```diff
@@ -0,0 +1,15 @@
+import Link from "next/link";
+
+export default function Navigation() {
+    return(
+       <footer>
+        <button>
+            <Link href="./index">Home</Link>
```

Comment:
You can just set the href to `href="/"` alternatively. In case you move your index.js file, the Link would still work.

Reply:
Thanks for the tip! Changed it.